lib/screen/home/home_screen.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:event_bloc/event_bloc.dart';
+import 'package:flutter/material.dart';
+
+import 'package:simple_exercise_builder/screen/home/selected_workout.dart';
+import 'package:simple_exercise_builder/screen/home/start_workout.dart';
+
+class HomeScreen extends StatelessWidget {
```

Comment:
I'd advise against naming your files by class names. It makes it look too much like Java. Hahahaha.